docs/index.md

```diff
@@ -190,3 +190,246 @@ The last step is to calculate all the specified corrections for the main fake fa
+          max: 150.0
+          rounding: 2
+      n_bins:  # this part is necessarily only needed for continuous variables
+        njets: 3
+      var_dependence_n_bins: [7, 5, 5]
+    var_dependence: pt_2
+    var_bins: {} # This will be filled by the script
+```
+
+The `equipopulated_binning_options` block has the following structure:
+
+| Parameter | Type | Description |
+|---|---|---|
+| `variable_config` | `dict` | Defines parameters for variables used in binning. See sub-parameters below. |
+| `n_bins` | `dict` | Specifies the desired number of bins per variable in `split_categories` if those are not set there in case of discrete variables, i.e. four equidistant bins in `pt_1`: `pt_1: 4` |
+| `var_dependence_n_bins` |  `int` or `list[int]` or `list[list[int]]` | Number of bins used for the variable defined in `var_dependence`, either as int (used by all categories) or a (nested) list of integers defining number of bins per (nested) category created.|
+
+The `variable_config` for each variable contains:
+
+| Sub-parameter | Type | Description |
+|---|---|---|
+| `min` | `float` | The lower bound for the variable's range. The first bin edge will be set to this value. This cut is applied calculation of equipopulated binning starts.  |
+| `max` | `float` | The upper bound for the variable's range. The last bin edge will be set to this value. This cut is applied calculation of equipopulated binning starts. |
+| `rounding` | `int` | The number of decimal places to which the calculated bin edges will be rounded to and written into the configuration file. |
+
+
+### Advanced Splitting Strategies
+
+The script supports various one- and two-dimensional splitting strategies, determined by the structure of `split_categories`. The order of variables defines the nesting hierarchy.
+
+1. One-Dimensional Splitting
+
+<ul>
+<li>
+<details>
+<summary><strong>By a discrete variable</strong> (e.g., <code>njets</code>): Categories are explicitly listed.</summary>
+
+Number of bins of `var_dependence` in each category are set explicitly
+```yaml
+split_categories:
+  njets: ["==0", "==1", ">=2"]
+split_categories_binedges:
+  njets: [-0.5, 0.5, 1.5, 12.5]
+equipopulated_binning_options:
+  variable_config:
+    pt_2:
+      min: 30.0
+      max: 150.0
+      rounding: 4
+  var_dependence_n_bins: [7, 5, 5]
+```
+Same number of bins of `var_dependence` in each category
+```yaml
+split_categories:
+  njets: ["==0", "==1", ">=2"]
+split_categories_binedges:
+  njets: [-0.5, 0.5, 1.5, 12.5]
+equipopulated_binning_options:
+  variable_config:
+    pt_2:
+      min: 30.0
+      max: 150.0
+      rounding: 4
+  var_dependence_n_bins: 7
+```
+</details>
+</li>
+<li>
+<details>
+<summary><strong>By a continuous variable</strong> (e.g., <code>pt_1</code>): An empty list <code>[]</code> is used as a placeholder.</summary>
+
+
+The script will first create equipopulated bins for pt_1 and then bin var_dependence within each of those new pt_1 categories.
```

Comment:
For consistency I would use `pt_1` and `var_dependence` instead of plain text. This comment goes also for the other examples